canonical/jaas-dashboard
# WD-619 - Upgrade to React 18

## Done

Upgrade to React 18.

## QA

- Pull code
- Run `dotrun clean && dotrun serve`
- Open http://0.0.0.0:8036/
- Check that the app works without errors.

## Details

Fixes: #1250.
